lib/tasks/install-blueprint.js

```diff
@@ -80,6 +113,30 @@ class InstallBlueprintTask extends Task {
     return execa('npm', ['install'], { cwd });
   }
 
+  _npmInstallModule(module, cwd) {
+    logger.info(`Running "npm install ${module}" in "${cwd}" ...`);
+    return execa('npm', ['install', module], { cwd })
+      .then(() => path.join(cwd, 'node_modules', module));
+  }
+
+  _handleNpmInstallModuleError(error) {
+    let match = error.stderr && error.stderr.match(NOT_FOUND_REGEXP);
+    if (match) {
+      let packageName = match[1];
+      throw new SilentError(`The package '${packageName}' was not found in the npm registry.`);
+    }
+
+    throw error;
+  }
+
+  _validateNpmModule(modulePath, packageName) {
+    logger.info(`Checking for "ember-blueprint" keyword in "${packageName}" module ...`);
+    let pkg = require(path.join(modulePath, 'package.json'));
```

Comment:
I know, but it's probably faster since the list of extensions won't have to be tried sequentially, right?